charts/gsp-cluster/values.yaml

```diff
@@ -135,18 +135,638 @@ fluentd-cloudwatch:
     requests:
       memory: 512Mi
   image:
-    tag: v1.3.2-debian-cloudwatch  # More recent image needed for fluentd-kubernetes-daemonset to avoid utf-8 encoding errors
+    tag: v1.7.4-debian-cloudwatch-1.0
```

Comment:
The chart default is `v1.7.3-debian-cloudwatch-1.0` - is there a reason we need `1.7.4`? If not I'd rather remove this line and allow the chart to keep us up to date.

Reply:
Done.